src/rprblender/nodes/node_parser.py

```diff
@@ -225,6 +225,24 @@ def create_arithmetic(self, op_type, color1, color2=None, color3=None):
 
         return rpr_node
 
+    def get_generated_UV(self):
```

Comment:
More correctly to move this method into NodeParser class from BaseNodeParser, because it uses self.node_item() function which is undefined in BaseNodeParser.